frontend/src/pages/user/seller/info/index.tsx

```diff
@@ -0,0 +1,151 @@
+import { useForm, SubmitHandler } from 'react-hook-form';
+import { Col, Row } from 'react-bootstrap';
+import TButton from '@components/TButton';
+import { useRef, useState } from 'react';
+
+interface ShopInfoProps {
+  shopName: string;
+  shopIcon: File | undefined;
+  visibility: boolean;
+  description: string;
+}
+
+const SellerItemStyle = {
+  borderRadius: '10px',
+  padding: '10% 10% 5% 10%',
+  border: '1px solid var(--button_border, #34977F)',
+  background: 'var(--button_dark, #135142)',
+};
+
+const userImgStyle = {
+  borderRadius: '50%',
+  width: '100px',
+  height: '100px',
+  cursor: 'pointer',
+};
+
+const labelStyle = {
+  fontSize: '16px',
+  fontWeight: '500',
+  color: '#FFFFFFAA',
+};
+
+const SellerInfo = () => {
+  //TODO: get the info from existing shop
+  const { register, handleSubmit, watch, setValue } = useForm<ShopInfoProps>({
+    defaultValues: {
+      shopName: 'shop name',
+      shopIcon: undefined,
+      visibility: false,
+      description: 'shop description',
+    },
+  });
+  const OnFormOutput: SubmitHandler<ShopInfoProps> = (data) => {
+    console.log(data);
+    return data;
+  };
+
+  // icon upload thing
+  const [image, setImage] = useState<string | undefined>(undefined);
+  const iconOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (e.target.files) {
+      //TODO? check file type
+      setValue('shopIcon', e.target.files[0]);
+      setImage(URL.createObjectURL(e.target.files[0]));
+    }
+  };
+  const hiddenFileInput = useRef<HTMLInputElement>(null);
+  const handleIconClick = () => {
+    if (hiddenFileInput.current) {
+      hiddenFileInput.current.click();
+    }
+  };
+
+  return (
+    <div>
+      <div className='title'>Shop info</div>
+      <hr className='hr' />
+      <form onSubmit={handleSubmit(OnFormOutput)}>
+        <Row>
+          {/* left half */}
+          <Col xs={12} md={6}>
+            <Row className='center' style={{ height: '100%' }}>
+              <Col />
+              <Col xs={8} xl={7}>
+                <div style={SellerItemStyle}>
+                  {/* upload img */}
+                  <div className='center'>
+                    <input
+                      type='file'
+                      onChange={iconOnChange}
+                      style={{ display: 'none' }}
+                      ref={hiddenFileInput}
+                    />
```

Comment:
currently, i can't upload new image for shop in `/user/seller/info`, is it normal?

Reply:
do u click the shop icon?

Reply:
sorry my bad, no problem now